curr_scraper/settings.py

```diff
@@ -11,8 +11,10 @@
 """
 
 from pathlib import Path
+from dotenv import load_dotenv
 import os
 import sys
+load_dotenv()
```

Comment:
Generally, it's not a good idea to import environment variables directly from .env, because its still not very secure. In production, you usually use secure storage like GithHub secrets, AWS SSM Parameters storage, etc... These storages import data directly into environment variables.

Reply:
Got it, will use more secure storage as needed.

Reply:
@Fazzer6132 You are still using load_dotenv in this PR, please remove it.
For local development, there are plenty of ways to define Environment variables other then load_dotenv

Reply:
@Rolloniel Should I use a secret manager or just a more secure way to load local .env file?